### PR TITLE
Create visually appealing Navbar

### PR DESCRIPTION
**Pull request by fredrir**
image
image

### CODE REVIEW

**Comment by fredrir** [comment]
What you think @julian-ao ?

File: components/Navbar.tsx [hunk 1]
```diff
@@ -1,62 +1,147 @@
...
-          href="/kontakt"
-          className="px-4 py-2 bg-onlineBlue text-white rounded-full hover:bg-opacity-70 transition-colors"
-        >
-          Ta kontakt
-        </Link>
-        <DropdownMenu /> {/* TODO: fix mobile menu */}
-      </motion.nav>
+    <header
+      ref={navRef}
+      className="fixed top-0 left-0 right-0 z-50 text-gray-100 bg-transparent/70"
```
Comment (julian-ao) [inline components/Navbar.tsx]: Hva med å gjøre bakgrunnen litt blurry?
image
```suggestion
      className="fixed top-0 left-0 right-0 z-50 text-gray-100 bg-transparent/70 backdrop-blur-sm"
```

**Review by julian-ao — APPROVED** [review]
image

File: components/Navbar.tsx (hunk 1 above)
Comment (fredrir) [reply components/Navbar.tsx]: 🔥🔥

**Pull request merged**